.github/workflows/test.yaml

```
@@ -82,6 +88,12 @@ jobs:
                   "python-version": "3.11",
                   "os": "ubuntu-latest"
                 }
+              ],
+              "exclude": [
+                {
+                  "python-version": "3.8",
+                  "os": "macos-latest"
+                },
```

```suggestion
                }
```

I think you are at lunch so I will just update these.